vcs/src/main/resources/ext.py

```diff
@@ -22,9 +22,13 @@
 import mercurial
 import mercurial.patch
 import mercurial.mdiff
+import mercurial.util
 import difflib
 import sys
 
+# space separated version list
+testedwith = '4.9.2 5.0.2'
+
```

Comment:
Does a space separated string named `testedWith` mean something special in Mercurial? Otherwise I would probably just have this a comment instead of a variable (or if you _really_ want a variable, why not use a `set`?)

Reply:
It's used to mark supported versions per: https://www.mercurial-scm.org/wiki/WritingExtensions#Marking_compatible_versions

Reply:
Not super needed, but the back trace I got earlier from the extension didn't include the supported versions. This adds the supported version to the backtrace e.g.:

```
** Unknown exception encountered with possibly-broken third-party extension log-git
** which supports versions 4.9 of Mercurial.
```

Seemed nice to have...

Reply:
Ok, sounds good, lets keep it then :+1: